api/models/User.js

```diff
@@ -32,7 +32,10 @@ module.exports = {
     email: "string",
     firstname: "string",
     lastname: "string",
-    intro: "text",
+    intro: {
```

Comment:
Does this break if someone already has one longer than 10k? If you could run the code before this change, add one with longer than 10k, then run this code and see if there's an issue. If so (probably is), we'll need to run a database migration if anyone has one longer (I bet there is at least 1)

Reply:
Ok so I tested it and after switching to this branch I received the following error:
![2018-02-19 19_17_45-xubuntu64 uruchomiona - oracle vm virtualbox](https://user-images.githubusercontent.com/15113729/36392793-2276d56c-15ad-11e8-8688-abd94c7fa0d8.png)

From user perspective nothing happened and when I tried to edit (but still have more than 10k) nothing happened:
![2018-02-19 19_19_48-xubuntu64 uruchomiona - oracle vm virtualbox](https://user-images.githubusercontent.com/15113729/36392829-4e81a1e6-15ad-11e8-8973-5027d78a00dc.png)

Once I shortened it, it got changed.

And after adding word-break:
![2018-02-19 19_26_44-xubuntu64 uruchomiona - oracle vm virtualbox](https://user-images.githubusercontent.com/15113729/36392862-7dca7c48-15ad-11e8-821f-883e6f569a84.png)

Reply:
Commenting just to say that @SnowPhoenix checked and no one has >10k anyway, so we shouldn't need to worry (although we should double check before deploying)